### PR TITLE
Gh98 fix failing ci

### LINKED ISSUE #98: Fix Failing CI Builds
### Cookbook version
v1.0.5/master

### Chef-client version
14.4.7

### Platform Details
Travis-CI

### Steps to Reproduce:
Running master on Travis-CI

### Expected Result:
CI builds show be passing.

### Actual Result:
Builds are failing on Travis-CI

### PR DESCRIPTION
**Pull request by pwelch**
### Description

Right now Test Kitchen is just looking for an enabled service but
there seems to be an issue with the test VMs and enabling the service
without a reboot.

This can be solved in another change, for now, just verify that the
service was installed. The test coverage will also need to be expanded
in the future.

### Issues Resolved

Fixes #98

### Check List
- [ ] All tests pass. See https://github.com/chef-brigade/sysctl/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -12,8 +12,6 @@ verifier:
 platforms:
   - name: centos-6.9
```
Comment (damacus) [inline .kitchen.yml]: These platforms can be just the major versions now. Take a look at postgresql for the format 👍

File: metadata.rb [hunk 2]
```diff
@@ -6,7 +6,8 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '1.0.5'
 
-depends 'compat_resource', '>= 12.14.6'
+chef_version '>= 12.19'
```
Comment (damacus) [inline metadata.rb]: We're depreciating 12 now as it's EOL

**Comment by pwelch** [comment]
@damacus Thanks for the feedback! Added suggested changes.

File: .kitchen.yml [hunk 3]
```diff
@@ -10,11 +10,9 @@ verifier:
...
-  - name: centos-6.9
-  - name: centos-7.3
-  - name: debian-7.11
-    run_list: apt::default
-  - name: debian-8.7
+  - name: centos-6
+  - name: centos-7
+  - name: debian-8
     run_list: apt::default
   - name: fedora-25
```
Comment (damacus) [inline .kitchen.yml]: fedora 28 is available

**Comment by pwelch** [comment]
@damacus Added feedback

**Comment by damacus** [comment]
Boom! Thanks!

**Pull request merged**